package.json

```diff
@@ -34,6 +34,7 @@
     "@types/lodash.camelcase": "^4.3.9",
     "@types/node": "^22.10.7",
     "@types/pretty-ms": "^5.0.1",
+    "@vitest/coverage-v8": "3.0.5",
```

Comment:
We're adding this but not actually using it? We need to either pass the `--coverage` flag to Vitest or have a `vitest.config.ts` file which enables coverage.

Reply:
Yeah this was so that `yarn test --coverage` worked